resources/css/dropdown.css

```diff
@@ -40,6 +40,7 @@
   padding: 4px 12px;
   border: none;
   border-radius: 0;
+  font-size: 12px;
```

Comment:
Note that this is a global component and will not be only used in the menu. Was it intentional to change it globally or just for the manu?

Reply:
It wasn't intentional to modify the dropdowns globally, but only those in the navbar. I only found this class in the navbar ones, so I made this modification directly through it. By the way, if the goal of keeping the font at this larger size is accessibility, then I think it's fair to maintain it, at first, I thought it was a silly mistake.